config/config.json

```diff
@@ -41,6 +41,7 @@
         "PostEditTimeLimit": 300,
         "TimeBetweenUserTypingUpdatesMilliseconds": 5000,
         "EnableUserTypingMessages": true,
+        "EnableUserTypingMessages": true,
```

Comment:
@mikaoelitiana can you help remove this change in config.json?

Our fault, we originally had this setting duplicated in config.json which got fixed in `master`.